hack/app_sre_create_image_catalog.sh

```diff
@@ -16,14 +16,14 @@ rm -rf "$SAAS_OPERATOR_DIR"
 
 git clone \
     --branch $BRANCH_CHANNEL \
-    https://app:${APP_SRE_BOT_PUSH_TOKEN}@github.com/app-sre/saas-hive-operator-bundle.git \
+    https://app:${APP_SRE_BOT_PUSH_TOKEN}@gitlab.cee.redhat.com/service/saas-hive-operator-bundle.git \
     $SAAS_OPERATOR_DIR
 
 # remove any versions more recent than deployed hash
 REMOVED_VERSIONS=""
 if [[ "$REMOVE_UNDEPLOYED" == true ]]; then
     DEPLOYED_HASH=$(
-        curl -s 'https://raw.githubusercontent.com/app-sre/saas-hive/master/hive-services/hive.yaml' | \
+        curl -s 'https://gitlab.cee.redhat.com/service/saas-hive/raw/master/hive-services/hive.yaml' | \
```

Comment:
This link will not be accessible to folks outside of RH network. Is it ok to keep links specific to RH in this repository?

Reply:
Would like to get all of this out of our repo at some point, but in meantime I don't have any objection to internal URLs showing up in these internal env only files.

Reply:
I think the question is if these links wont stay here then where? Where is the line we need draw? IMO it is not priority to get these answers right now, but we should think about these and find an answer.